charts/cluster-api-cluster-openstack/templates/workload.yaml

```diff
@@ -107,6 +107,16 @@ spec:
         path: {{ $file.path }}
         permissions: "0600"
       {{- end }}
+      - path: /etc/kubernetes/patches/kubeletconfiguration0+strategic.json
+        owner: "root:root"
+        permissions: "0644"
+        content: |
+            {
```

Comment:
You know what, I think it'll make life easier for the whole world if we just say...

`values.yaml`

```yaml
workloadPools:
  kubelet:
    serializeImagePulls: "false"
    foo:
      bar: sheep
      baz: ball
```

Then go something like...

```yaml
{{- with $config := .Values.workloadPools.kubelet }}
{{- $_ := set $config "apiVersion" "kubelet.config.k8s.io/v1beta1" -}}
{{ $_ := set $config "kind" "KubeletConfiguration" -}}
...
content: {{ toJson $config }}
{{- end }}
```

Then you can inject anything ever without another modification just using `--set`